main.py

```diff
@@ -1,62 +1,13 @@
-import pygame
-import Utils.Map as mp
+import Utils.Game
 
-WIDTH, HEIGHT = 800, 800
-ROW, COLUMN = 8, 8
-SIZE = WIDTH // ROW
-IMAGES = {}
-# Define colors
-WHITE = (255, 255, 255)
-GRAY = (128, 128, 128)
+# TODO: I found some bugs:
```

Comment:
Instead leaving random comments in the code - open a ticket on your github board with description of an issue. That would be much more beneficial for everyone